packages/react/src/components/UIShell/SideNav.js

```diff
@@ -119,14 +123,21 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
     eventHandlers.onMouseLeave = () => handleToggle(false, false);
   }
 
+  useEffect(() => {
+    if (expanded && !isFixedNav) {
+      sideNavRef.current.focus();
+    }
```

Comment:
let's check for undefined just in case
```suggestion
    if (expanded && !isFixedNav && sideNavRef.current) {
      sideNavRef.current.focus();
    }
```